src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-topic-details/rgw-topic-details.component.html

```diff
@@ -0,0 +1,84 @@
+  <ng-container *ngIf="!!selection">
+    <cds-tabs type="contained"
+              theme="light">
+      <cds-tab heading="Destination Endpoints"
+               i18n-heading>
+        <table class="cds--data-table--sort cds--data-table--no-border cds--data-table cds--data-table--md"
+               data-testid="rgw-topic-details">
+          <tbody>
+            <tr>
+              <td i18n
+                  class="bold">Push endpoint args</td>
```

Comment:
Are they called args or "arguments" ?
I would advice to chekc all terms here with doc team , thanks